apiclient/account.go

```diff
@@ -107,6 +107,16 @@ func (c *HTTPclient) AccountBootstrap(faucetPkg *models.FaucetPackage, metadata
 		metadataURI = "ipfs://" + ipfs.CalculateCIDv1json(metadataBytes)
 	}
 
+	if sik == nil {
+		sikSign, err := c.account.SignEthereum([]byte(DefaultSIKContent))
```

Comment:
I think the crypto/ethereum package could be extended to have a method that returns and abstracts the SIK generation. So we can do `c.account.Sik()`.